src/main/webapp/dashboard/view/Master.view.xml

```diff
@@ -33,6 +33,8 @@
                                                press="onAdd" tooltip="{i18n>AddNewConnector}"/>
                         <OverflowToolbarButton icon="sap-icon://sort" type="Transparent"
                                                press="onSort" tooltip="{i18n>Sort}"/>
+                        <OverflowToolbarButton icon="sap-icon://edit" type="Transparent"
+                                               press="onChange" tooltip="{i18n>Change}"/>
```

Comment:
Please also rename "Change" to "Edit" everywhere.

Reply:
In general we should try to follow the SAP Fiori Design guidelines:
https://experience.sap.com/fiori-design-web/object-page/

So like Beatriz said they also call it edit:
https://experience.sap.com/fiori-design-web/object-page/#global-edit